tests/e2e/expected_results/purl/npm/semver/semver.json

```diff
@@ -89,62 +93,14 @@
                         }
                     }
                 ],
-                "npm Registry": [
```

Comment:
Any idea why the provenance is removed from the JSON report? It might not be related to this PR but I wonder what caused it.

Reply:
Looks like it is caused by the `provenance_available` check modification that was done to allow it to return early if provenance was already found. If that happens the release assets are not collected, so this will report as being empty.

Reply:
Thanks for looking into the issue. I guess once we move all the provenance finding code from the check to the new `ProvenanceFinder.find_provenance`  method, this issue will be resolved.